x-pack/legacy/plugins/index_lifecycle_management/public/sections/policy_table/components/policy_table/policy_table.js

```diff
@@ -390,6 +406,18 @@ export class PolicyTable extends Component {
       } else if (totalNumberOfPolicies > 0) {
         tableContent = (
           <EuiTable className="policyTable__horizontalScroll">
+            <caption
+              role="status"
+              aria-relevant="text"
+              aria-live="polite"
+              className="euiScreenReaderOnly"
```

Comment:
Is it possible to use the [EuiScreenReaderOnly](https://elastic.github.io/eui/#/utilities/accessibility) component here instead of using the CSS class? The comment applies in other places this classname is used.

Reply:
yep, good call. fixed.